Slims/slims.py

```diff
@@ -0,0 +1,183 @@
+import os
```

Comment:
It's probably a good idea to rename the folder called `Slims` here. It might cause import issues with the actual slims api python package. I would suggest `api` or `tools` or something akin.